lua/lv-utils/init.lua

```diff
@@ -1,5 +1,61 @@
 local lv_utils = {}
 
+-- recursive Print (structure, limit, indent)
+local function rPrint(s, l, i)
```

Comment:
Even though you added a note above, I still think one char variable name is bad because it makes the rest of the code harder to read

Reply:
agreed, gonna fix it asap :running_man: